opentelemetry-api/src/opentelemetry/context/default_context.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import typing
+
+from opentelemetry.context.context import Context, RuntimeContext
+
+
+class DefaultRuntimeContext(RuntimeContext):
+    """A default implementation of the RuntimeContext interface using
+    a dictionary to store values.
+    """
+
+    def __init__(self) -> None:
+        self._current_context = None  # type: typing.Optional[Context]
+
+    def set_current(self, context: Context) -> None:
+        """See `opentelemetry.context.RuntimeContext.set_current`."""
+        self._current_context = context
```

Comment:
The other implementation of the `RuntimeContext` all return None, the only place where this is different is in the `context` module where `set_current` returns a `Context`, think it makes sense to keep the logic of returning a handle in a single place, but I could be convinced otherwise.

Reply:
You're right, not sure what I was looking at.
An advantage of returning the handle on each implementation would be that the mechanishm underneath could be used, for instance directly returning the handle returned by [contextvars.set](https://docs.python.org/3/library/contextvars.html#contextvars.ContextVar.set). Anyway, I consider it is a small detail and would mark this conversion as resolved.